webhook/resourcesemantics/defaulting/defaulting_test.go

```diff
@@ -370,11 +455,13 @@ func createCreateResource(ctx context.Context, t *testing.T, r *Resource) *admis
 
 func TestAdmitUpdates(t *testing.T) {
 	tests := []struct {
-		name      string
-		setup     func(context.Context, *Resource)
-		mutate    func(context.Context, *Resource)
-		rejection string
-		patches   []jsonpatch.JsonPatchOperation
+		name                     string
+		setup                    func(context.Context, *Resource)
+		mutate                   func(context.Context, *Resource)
+		createResourceFunc       func(name string) *Resource
+		createUpdateResourceFunc func(ctx context.Context, t *testing.T, old, new *Resource) *admissionv1.AdmissionRequest
```

Comment:
Since the setup is very different it might make sense to split the callback tests into a separate `TestX(...)` function